libs/wingc/src/jsify/snapshots/capture_type_inflight_class_sibling_from_init.snap

```diff
@@ -39,11 +39,11 @@ module.exports = function({  }) {
       class Bar {
         async func() {
           x;
-          new Foo();
+          (await (async () => {const o = new Foo(); if ('$inflight_init' in o) { await o.$inflight_init(); } return o; })());
```

Comment:
```suggestion
           await new.Foo()?.$inflight_init();
```

Or something along those lines.

Reply:
This won't work :( because we need to return the new object and the optional chaining here might return `undefined`.

Reply:
Resolving this, see open [conversation above](https://github.com/winglang/wing/pull/4390#discussion_r1348000429)